### ISSUE TITLE
Crash when pasting symbols

### ISSUE THREAD

**Issue by jonarnoldmusic** [musescore/MuseScore#22711]
### Issue type

Crash or freeze

### Bug description

When pasting symbols of an incipit into a specific staff, the program crashes. Pasting into other parts worked OK.

### Steps to reproduce

1. Copy content from incipit in Discantus I (top staff).
2. Select incipit measure for Tenor 1 (fourth staff down).
3. Paste.

### MuseScore Version

4.3

### Regression

I don't know

### Operating system

Windows 11

### Additional context

[MuseScore_240507_150738.log](https://github.com/musescore/MuseScore/files/15239894/MuseScore_240507_150738.log)
[MSCZproject.zip](https://github.com/musescore/MuseScore/files/15239909/MSCZproject.zip)

**Comment by bkunda** [comment]
I've had a quick look and it does seem oddly isolated to that tenor 1 staff (as you point out, pasting symbols into other incipit staves does not crash). 
It looks like this score was created in 4.3, so I'm not sure we can check the regression status of this one (unless you happen to still have a copy saved in a previous version of MuseScore?).

**Comment by Jojo-Schmitz** [comment]
IIRC by setting some development option you can read 4.3 scores in 4.2

**Comment by DmitryArefiev** [comment]
> IIRC by setting some development option you can read 4.3 scores in 4.2

Yes, we added that in 4.2.1 (and score can be re-saved as 4.2 score)

The **crash happens** in **4.2.0** as well

**Comment by jonarnoldmusic** [comment]
There are some other oddities about this file: I discovered that every symbol I insert from the master pallette into the incipit measure is inserted twice (near the beginning and at the ending barline). I couldn't replicate this with a fresh file.

**Comment by jonarnoldmusic** [comment]
I copied the incipits and score setup from this attached file. Not sure which version it was last saved in, but it's before 4.3.
[OrigMusescoreFile.zip](https://github.com/musescore/MuseScore/files/15250757/OrigMusescoreFile.zip)
I'm able to paste into all staves, including the fourth one (labelled Bassus), so something became corrupted somehow. I think I created the file in the bug report by saving a copy of this file, deleting the musical content, and pasting content from a separate midi import after the incipit.

**Comment by FrancRos31** [comment]
Cannot reproduce anymore.

https://github.com/user-attachments/assets/e8e3dad5-3955-45b1-8338-0b6fc673f085

OS: Windows 10 Version 22H2 or later, Arch.: x86_64, MuseScore Studio version (64-bit): 4.6.0-252320405, revision: github-musescore-musescore-6509a9e

**Comment by ghost** [comment]
Me neither. Will close this (@jonarnoldmusic please feel free to check yourself in 4.5 or later, and if you manage to trigger the crash we can reopen it)
